steward_proto/proto/cellars/uniswapv3.proto

```diff
@@ -28,3 +28,15 @@ message RebalanceResponse {}
 //     "check_frequency": 3600,
 //     "schema_type": "UNIv3",
 // }
+
+service UniswapV3DirectCellar {
```

Comment:
We don't need to implement a proto -- steward is either going to be running in Cosmos voting mode or single signer mode, in either case it can expose the same endpoint and receive the same proto messages as the Cosmos mode, because we do not expect the data provider to adjust their format.

Reply:
Does this means there shouldn't be a `DirectRebalance` endpoint and a `UniswapV3DirectCellar`? So, I should go ahead with the `UniswapV3CellarAllocator` service?

Reply:
Well, you still want `UniswapV3DirectCellar`. I think you would do this by using `UniswapV3CellarAllocatorServer` when doing `add_service`, but you instead pass it `UniswapV3DirectCellar` which is also an implementation of `server::UniswapV3CellarAllocator` using the `UniswapV3DirectCaller` struct, similar to the cosmos version. This way you are adhering to the same API but can change the internal implementation.

Reply:
I get an error if I do it this way.

```the trait bound `cellars::uniswapv3::UniswapV3DirectCellar: steward_proto::uniswapv3::uniswap_v3_cellar_allocator_server::UniswapV3CellarAllocator` is not satisfied
the trait `steward_proto::uniswapv3::uniswap_v3_cellar_allocator_server::UniswapV3CellarAllocator` is not implemented for `cellars::uniswapv3::UniswapV3DirectCellar`
```